test/parallel/test-https-deprecated-urls.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable node-core/crypto-check */
```

Comment:
I don't believe this comment is needed.

Reply:
You should also be able to remove it from the other test. It's only needed if the test file tests both http and https. If it's just one or the other, then it's not needed.